### PR TITLE
adding after selector to gbs_boom style

### PR DESCRIPTION
**Pull request by luchotc**
Resolves https://github.com/mumuki/mumuki-gobstones-runner/issues/138

https://stackoverflow.com/questions/31636402/why-is-css-content-image-not-displaying

### CODE REVIEW

File: dist/gs-board.html [hunk 1]
```diff
@@ -1 +1 @@
...
...t?t.base:void 0,n=null!=i?i.base:void 0,null!=o&&null!=r)return null!=n&&n.enabled?this._setBorderOff():this._setBorderOn(),this._updateBorderImages(n),this.updateStyles(),this.querySelectorAll("gs-cell").forEach(function(e){return e.updateCellStyles(o,r,n)})},_setBorderOn:function(){return this.customStyle["--cell-padding"]="2px",this.customStyle["--cell-border"]="solid #888 1px"},_setBorderOff:function(){return this.customStyle["--cell-padding"]="0 0",this.customStyle["--cell-border"]="none"},_updateBorderImages:function(e){var t;return t=function(t){return function(t){var i,n;if(i=null!=e&&null!=(n=e.borders)?n[t]:void 0,(null!=e?e.enabled:void 0)&&null!=i)return"url("+i+")"}}(),this.customStyle["--top-left-background-url"]=t("topLeft"),this.customStyle["--top-right-background-url"]=t("topRight"),this.customStyle["--bottom-left-background-url"]=t("bottomLeft"),this.customStyle["--bottom-right-background-url"]=t("bottomRight"),this.customStyle["--left-background-url"]=t("left"),this.customStyle["--top-background-url"]=t("top"),this.customStyle["--right-background-url"]=t("right"),this.customStyle["--bottom-background-url"]=t("bottom")}});</script></dom-module></div></body></html>
```
Comment (luchotc) [inline dist/gs-board.html]: This generated file has a difference: `assetpath="dist/components/"`

Suspicious :thinking:. Is this ok? @rodri042

**Comment by luchotc** [comment]
Initially, I'd just added the after selector for Firefox to work. But after updating my version of Firefox, it wasn't working either :confounded: 
The only way to make it work for both browsers was to replace it with a div element.

**Comment by afska** [comment]
The code looks good. Does it work on Chrome as well? If so, I'll merge and
publish in a few hours

El 4 feb. 2019 3:50 p. m., "luchotc" <notifications@github.com> escribió:

Initially, I'd just added the after selector for Firefox to work. But after
updating my version of Firefox, it wasn't working either 😖
The only way to make it work for both browsers was to replace it with a div
element.

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/gobstones/gs-board/pull/20#issuecomment-460365580>, or mute
the thread

**Comment by luchotc** [comment]
Yep, it works in Chrome as well! Thanks :smile:

**Comment by afska** [comment]
Published as `1.18.0`

**Pull request merged**